PyTorch/Segmentation/MaskRCNN/pytorch/train_net_herring.py

```diff
@@ -232,9 +234,20 @@ def main():
         type=str,
         default=None
     )
+    parser.add_argument('--seed',
+                        '-s',
+                        help='manually set random seed for torch',
+                        type=int,
+                        default=99)
 
     args = parser.parse_args()
 
+    # Set seed to reduce randomness
+    random.seed(args.seed + herring.get_local_rank())
```

Comment:
I think just setting the seeds to `args.seed` might be better since the calling script can use `OMPI_COMM_WORLD_LOCAL_RANK`

Reply:
The PT BERT script used by NVIDIA does the similar thing: https://github.com/HerringForks/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT/run_pretraining.py#L499 .

So are you suggesting `random.seed(args.seed)` ?

Reply:
Should the scripts not use a fixed seed across the cluster rather than a seed dependent on it's rank ?

Reply:
@karan6181 yes. @piyushghai The user can set the per rank seeds to whatever he/she wants if we just pass the seed argument through to the seed functions.

Reply:
From what I have observed, TF and MX uses a same seed value across the cluster. However, only PT uses the seed based on ranks. I am also inclined towards using the same seed value across the cluster unless someone have a conflict reason!!

Reply:
This is how NVIDIA forks uses seed in PyTorch:
https://github.com/HerringForks/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT/run_pretraining.py#L499
https://github.com/HerringForks/DeepLearningExamples/blob/master/PyTorch/Detection/SSD/main.py#L151

In both cases, they use ranks